examples/column_view_datagrid/main.rs

```diff
@@ -0,0 +1,105 @@
+mod grid_cell;
+
+use crate::grid_cell::Entry;
+use crate::grid_cell::GridCell;
+use gtk::gio;
+use gtk::glib::BoxedAnyObject;
+use gtk::prelude::*;
+
+struct Row {
+    col1: String,
+    col2: String,
+}
+
+use std::cell::Ref;
+
+fn main() {
+    let app = gtk::Application::new(
+        Some("com.github.gtk-rs.examples.columnview-example"),
+        Default::default(),
+    );
+    app.connect_activate(build_ui);
+    app.run();
+}
+
+fn build_ui(application: &gtk::Application) {
+    let window = gtk::ApplicationWindow::builder()
+        .default_width(320)
+        .default_height(480)
+        .application(application)
+        .title("ColumnView Example")
+        .build();
+
+    let vbox = gtk::Box::new(gtk::Orientation::Vertical, 5);
+
+    let store = gio::ListStore::new(BoxedAnyObject::static_type());
+
+    let b1 = BoxedAnyObject::new(Row {
+        col1: "super".to_string(),
+        col2: "awesome".to_string(),
+    });
+    let b2 = BoxedAnyObject::new(Row {
+        col1: "extra".to_string(),
+        col2: "cool".to_string(),
+    });
+    let b3 = BoxedAnyObject::new(Row {
+        col1: "blazing".to_string(),
+        col2: "fast".to_string(),
+    });
+    store.append(&b1);
+    store.append(&b2);
+    store.append(&b3);
```

Comment:
You could create a vec with "super", "awesome",... and then iterate through it.

Reply:
You can directly append a slice

Reply:
made a new thing and it appends 10000 items just for fun, updated screenshot to reflect too:)